packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -86,7 +86,8 @@ class RefreshIndicator extends StatefulWidget {
     this.displacement: 40.0,
     @required this.onRefresh,
     this.color,
-    this.backgroundColor
+    this.backgroundColor,
+    this.acceptNotification: defaultScrollNotificationPredicate,
```

Comment:
i assume you don't want this one to be null either?

Reply:
(update the docs to be in a similar style to the other class, listing arguments that musn't be null)

Reply:
(and assert accordingly)

Reply:
Done